test2.txt

```diff
@@ -0,0 +1 @@
+cek
```

Comment:
delete this file. not used.

Reply:
same for tst.txt

Reply:
dah diapus https://github.com/AxlAdilla/Simpin/pull/1/commits/6ce3d56f2452a5147fe46f4f4c21e5e7a2aa2edc